var/spack/repos/builtin/packages/py-virtualenv/package.py

```diff
@@ -21,6 +22,18 @@ class PyVirtualenv(PythonPackage):
     version('1.11.6', sha256='3e7a4c151e2ee97f51db0215bfd2a073b04a91e9786df6cb67c916f16abe04f7')
 
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'), when='@20.10.0:')
 
     # not just build-time, requires pkg_resources
     depends_on('py-setuptools@40.6.3:', type=('build', 'run'))
+    depends_on('py-setuptools@41.00.03:', type=('build', 'run'), when='@20.10.0:')
+    depends_on('py-setuptools-scm@2:', type=('build', 'run'), when='@20.10.0:')
+
+    depends_on('py-backports-entry-points-selectable @1.0.4:', type=('build', 'run'), when='@20.10.0:')
```

Comment:
Is this only needed for Python < 3.10? If so, it might be useful to write `when=@20.10.0: ^python@:3.9`

Reply:
@scheibelp https://github.com/pypa/virtualenv/blob/20.10.0/setup.cfg#L43 - no condition there, so I'd prefer not to add it here. But if you insist...

Reply:
I don't think it's mandatory: I was just basing this off of reading the github page for py-backports-entry-points-selectable (I usually look for some upper bound constraint on any `*backports*`-type package). I can approve/merge as-is.

Reply:
Let's merge as-is